pkg/bootstrapper/bootstrapper.go

```diff
@@ -314,6 +337,9 @@ func (wmcb *winNodeBootstrapper) createKubeletService() error {
 	if cloudConfigValue, ok := wmcb.kubeletArgs[cloudConfigOption]; ok {
 		kubeletArgs = append(kubeletArgs, "--"+cloudConfigOption+"="+cloudConfigValue)
 	}
+	if nodeWorkerLabel, ok := wmcb.kubeletArgs["node-labels"]; ok {
```

Comment:
I missed this while pushing my changes. The fact that we're appending to map and then to a slice is kind of bothering. I am thinking of writing a function which converts the map to slice, so that we don't have to update at 2 different places.